vscode/src/notebookTemplate.ts

```diff
@@ -111,7 +116,35 @@ plt.title("RandomBit() Results")
 plt.bar(xlabels, counts)
 plt.xticks(xlabels, values)
 plt.show()
-      `,
+`,
+    },
+    {
+      kind: vscode.NotebookCellKind.Markup,
+      languageId: "markdown",
+      value: `## Q# widgets
+
+You can also use the \`qsharp_widgets\` package to visualize data. Install with \`pip install qsharp-widgets\``,
+    },
+    {
+      kind: vscode.NotebookCellKind.Code,
+      languageId: "python",
+      value: `from qsharp_widgets import Histogram
+
+Histogram(results)
+`,
+    },
+    {
+      kind: vscode.NotebookCellKind.Code,
+      languageId: "python",
+      value: `from qsharp_widgets import EstimatesPanel
+
+estimate = qsharp.estimate("RandomBit()", [
+    {"errorBudget": 0.333, "qubitParams": {"name": "qubit_gate_ns_e3"}},
+    {"errorBudget": 0.333, "qubitParams": {"name": "qubit_gate_us_e4"}},
+    {"errorBudget": 0.333, "qubitParams": {"name": "qubit_maj_ns_e6"}, "qecScheme": {"name": "floquet_code"}}
+])
+EstimatesPanel(estimate)
+`,
     },
```

Comment:
Does it really make sense to have a resource estimation sample that returns almost no usage? `RandomBit` doesn't have any T gates, CCNOT gates, or rotations, so the counts won't really show anything interesting.

Reply:
I didn't want to clutter it by adding a more complex algorithm. The notebook is just intended to show API usage. But yeah, could add something more interesting later. (Any ideas for something that fits in a few lines in a cell?)

Reply:
No, for bare bones, "here is how you get started with this," I think it's fine then.